decidim-core/app/cells/decidim/upload_modal/modal.erb

```diff
@@ -1,25 +1,24 @@
-<div class="reveal upload-modal" id="<%= modal_id %>" data-reveal>
-  <div class="reveal__header">
-    <h3 class="reveal__title" data-addlabel="<%= button_label %>" data-editlabel="<%= button_edit_label %>"><%= label %></h3>
-    <button class="close-button" data-close aria-label="<%= t("layouts.decidim.upload_modal.close_modal") %>"
-      type="button">
-      <span aria-hidden="true">&times;</span>
-    </button>
-  </div>
+<%= decidim_modal id: modal_id, class: "upload-modal" do %>
+<div data-dialog-container>
+  <%= icon "upload-cloud-2-line" %>
+  <div>
```

Comment:
The view looks a bit cluttered on mobile:
![Mobile modal](https://user-images.githubusercontent.com/864340/199271160-2ef20c5b-b21d-4ffa-a9b6-18e4351a0b93.png)

I cannot review this in the design app as I don't have an account there, so I don't know if this is fixed there.